src/script/common/c_content.cpp

```diff
@@ -426,6 +426,20 @@ TileDef read_tiledef(lua_State *L, int index, u8 drawtype)
 			L, index, "tileable_horizontal", default_tiling);
 		tiledef.tileable_vertical = getboolfield_default(
 			L, index, "tileable_vertical", default_tiling);
+		std::string align_style;
+		if (getstringfield(L, index, "align_style", align_style)) {
+			if (align_style == "user")
+				tiledef.align_style = ALIGN_STYLE_USER_DEFINED;
+			else if (align_style == "world")
+				tiledef.align_style = ALIGN_STYLE_WORLD;
+			else
+				tiledef.align_style = ALIGN_STYLE_NODE;
+		} else {
+			bool world_aligned = false;
+			getboolfield(L, index, "world_aligned", world_aligned);
```

Comment:
Undocumented. IMO, this boolean check isn't required. Just set it to `NONE` by default and only accept string values.

Reply:
Oh, yet another compatibility (with the previous PR). Will drop now.
Should it throw on invalid values?

Reply:
A boolean was used in #6075 IIRC